linkerd-monitoring/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: prometheus
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "k8s-monitor"
+    heritage: "{{ .Release.Service }}"
+    prometheus: k8s
```

Comment:
`prometheus: k8s-monitor` if you want to match with our infrstructure Prometheus